### PR TITLE
feat(kubernetes): add more info about k8s-v2 loadbalancer

### PR DESCRIPTION
**Pull request by posox**
Issue: https://github.com/spinnaker/spinnaker/issues/2710

deck

### CODE REVIEW

File: app/scripts/modules/kubernetes/src/v2/loadBalancer/details/details.html [hunk 1]
```diff
@@ -73,6 +73,70 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
...
+            text="ingress.ip"
+            tool-tip="'Copy Ingress IP to clipboard'">
+          </copy-to-clipboard>
+        </dd>
+      </dl>
+      <dl ng-if="ctrl.loadBalancer.menifest.spec.loadBalancerIP">
+        <dt>Load Balancer IP</dt>
+        <dd>
+          <a target="_blank" href="//{{ctrl.loadBalancer.manifest.spec.loadBalancerIP}}">
+            {{ctrl.loadBalancer.menifest.spec.loadBalancerIP}}
```
Comment (lwander) [inline app/scripts/modules/kubernetes/src/v2/loadBalancer/details/details.html]: s/menifest/manifest

File: app/scripts/modules/kubernetes/src/v2/loadBalancer/details/details.html [hunk 2]
```diff
@@ -73,6 +73,70 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
...
+          </a>
+          <copy-to-clipboard
+            ng-if="ingress.ip"
+            class="copy-to-clipboard copy-to-clipboard-sm"
+            text="ingress.ip"
+            tool-tip="'Copy Ingress IP to clipboard'">
+          </copy-to-clipboard>
+        </dd>
+      </dl>
+      <dl ng-if="ctrl.loadBalancer.menifest.spec.loadBalancerIP">
```
Comment (lwander) [inline app/scripts/modules/kubernetes/src/v2/loadBalancer/details/details.html]: s/menifest/manifest

**Comment by sbwsg** [comment]
Doh, merged before @lwander reviewed.

**Comment by lwander** [comment]
No worries

**Comment by sbwsg** [comment]
I'll fix up those changes now and make a push

**Comment by posox** [comment]
@sbwsg I can fix these issues.

**Comment by sbwsg** [comment]
Already done: https://github.com/spinnaker/deck/pull/5487

**Comment by posox** [comment]
Thank you!

**Comment by sbwsg** [comment]
Also, thanks!  These changes look great

**Pull request merged**